examples/get-training-model-progress.py

```diff
@@ -1,14 +1,10 @@
-from indico import IndicoClient, IndicoConfig
+from indico import IndicoClient
 from indico.queries import GetModelGroup, GetTrainingModelWithProgress
 
 # The model group ID can be found on the review page of the indico platform
 model_group_id = 4305
 
-my_config = IndicoConfig(
-    host="app.indico.io", api_token_path="./path/to/indico_api_token.txt"
```

Comment:
mmm I wonder if we remove this line, I think for like 90% of customer install they need to pass a config so worth showing?

Reply:
I wanted to abstract all the redundant client config bits to the custom_client example

Reply:
I'm questionning if that's a good idea for these examples, they can right now copy them as is, change the few fields that they need and go vs knowing how to get the config into the client. It's more verbose for sure.

Reply:
I can put them back, no problem. But fwiw, I would argue that it's just as easy to look up the client example and copy that :woman_shrugging: